### ISSUE TITLE
[sub-issue] CRw: Color contrast in the badges (faceted page navigation) is insufficient

### ISSUE THREAD

**Issue by bit-pro-iew-eui** [swiss/styleguide#280]
![image](https://cloud.githubusercontent.com/assets/7510842/5356998/bc9b3510-7fa2-11e4-9a53-9c752d3792d3.png)

Color contrast in the badges is insufficient.
==> Use #454545 for numbers/texts in Badges
